pages/table/resizable-columns.page.tsx

```diff
@@ -106,6 +108,8 @@ export default function App() {
   const stickyHeader = urlParams.stickyHeader ?? false;
   const resizableColumns = urlParams.resizableColumns ?? true;
   const fullPage = urlParams.fullPage ?? false;
+  const withColumnIds = urlParams.withColumnIds ?? true;
+  const withSelection = urlParams.withSelection ?? false;
```

Comment:
Isn't this mix of true and false defaults is hard to follow? Any chances of making it more ergonomic?

Reply:
Will try